testing/correctness/apps/dummy_python/README.md

```diff
@@ -0,0 +1,72 @@
+# Sequence Window Python
```

Comment:
Should this readme still exist?

Reply:
I just removed it locally, will be gone with the next push.